app/views/organisation_relationships/remove_housing_provider.html.erb

```diff
@@ -0,0 +1,23 @@
+<% @organisation_to_remove_id = params["organisation_to_remove_id"] %>
+<%= form_with url: housing_providers_organisation_path(organisation_to_remove_id: @organisation_to_remove_id), method: "delete", local: true do |f| %>
+  <% if current_user.support? %>
+    <%= render partial: "organisations/headings", locals: { main: @organisation.name, sub: nil } %>
+    <%= render SubNavigationComponent.new(items: secondary_items(request.path, @organisation.id)) %>
+    <h2 class="govuk-visually-hidden">Remove Housing Provider</h2>
+  <% end %>
+  <% @housing_provider = Organisation.find(@organisation_to_remove_id) %>
```

Comment:
I'd move this into the controller. You won't need to do this at the top of the file, then:

```
<% @organisation_to_remove_id = params["organisation_to_remove_id"] %>
```